app/main.py

```diff
@@ -1,21 +1,42 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
-
+    def __init__(self, row, column):
+        self.row = row
+        self.column = column
+        self.is_alive = True
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
+    def __init__(self, start, end):
         # Create decks and save them to a list `self.decks`
-        pass
+        self.start = start
+        self.end = end
+        self.is_drowned = False
+        self.decks = []
+
+    def crate_decks(self, i, j):
```

Comment:
There is a typo in the method name `crate_decks`. It should be `create_decks` to correctly reflect its functionality.